x-pack/legacy/plugins/siem/public/components/formatted_bytes/index.tsx

```diff
@@ -12,9 +12,7 @@ import { useUiSetting$ } from '../../lib/kibana';
 
 export const PreferenceFormattedBytesComponent = ({ value }: { value: string | number }) => {
   const [bytesFormat] = useUiSetting$<string>(DEFAULT_BYTES_FORMAT);
-  return (
-    <>{bytesFormat ? numeral(value).format(bytesFormat) : numeral(value).format('0,0.[0]b')}</>
-  );
+  return <>{numeral(value).format(bytesFormat || '0,0.[0]b')}</>;
```

Comment:
I would remove the double pipe or check `||` as it looks like it is not needed since you are passing in `DEFAULT_BYTES_FORMAT`?

Reply:
Looks like we don't ever check for null/undefined in other places:
https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/siem/public/pages/detection_engine/components/signals/signals_utility_bar/index.tsx#L48

So I am assuming it is safe to remove.

Reply:
There is an edge case where the user has explicitly unset that setting: they would get raw values with no (manual) formatting. In practice this doesn't look great, but it's not terrible either: 

<img width="746" alt="Network_-_Kibana" src="https://user-images.githubusercontent.com/657252/71856175-f497b780-3097-11ea-8f76-fecec080f8c5.png">

I can totally get behind the 'do what I specify in the settings' behavior and remove that guard, but I wanted to mention this just in case.

Reply:
215f59c